doc/cephfs/fs-volumes.rst

```diff
@@ -1340,5 +1340,23 @@ set with this id was present in the database
 
   $ ceph fs quiesce fs1 sub1 sub2 sub3 --set-id="external-id" --if-version=0
 
+
+.. _disabling-volumes-plugin:
+
+Disabling Volumes Plugin
```

Comment:
Please detail that this ins't the first resort and its good to first disable async threads to assess if the file system experience is improved.

Reply:
That's very useful. Thanks, I'll add it.

Reply:
Done.